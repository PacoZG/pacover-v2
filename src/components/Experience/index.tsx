'use client'
import React, { useEffect } from 'react'
import WorkingExperience from '@/components/Experience/WorkingExperience'
import { useTranslations } from 'next-intl'

const Experience: () => React.JSX.Element = () => {
  const t = useTranslations('stickers')
  useEffect(() => {
    window.scrollTo(0, 0)
  }, [])

  return (
    <div className="page-layout">
      <div className="flex justify-center w-full py-8">
        <div className="sticky-header rounded-t-xl">{t('experience')}</div>
      </div>

      <WorkingExperience />
    </div>
  )
}

export default Experience
