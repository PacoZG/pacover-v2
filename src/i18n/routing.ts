/* istanbul ignore file */
import { defineRouting } from 'next-intl/routing'

export const routing = defineRouting({
  locales: ['en', 'es'],

  defaultLocale: 'en',
})
