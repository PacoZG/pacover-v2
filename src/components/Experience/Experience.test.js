import React from 'react'
import { I18nextProvider } from 'react-i18next'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Experience from '.'
import i18n from '../../utils/language/i18next.test'

describe('Testing', () => {
  test('Snapshot of the Experience component', () => {
    const rederedHeader = renderer
      .create(
        <BrowserRouter>
          <I18nextProvider i18n={i18n}>
            <Experience />
          </I18nextProvider>
        </BrowserRouter>
      )
      .toJSON()
    expect(rederedHeader).toMatchSnapshot()
  })
})
