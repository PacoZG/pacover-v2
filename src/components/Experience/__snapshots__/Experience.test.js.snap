// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Snapshot of the Experience component 1`] = `
<div
  className="page-layout"
>
  <div
    className="sticky-header rounded-t-xl"
  >
    WORK EXPERIENCE
  </div>
  <div
    className=" flex flex-col items-center justify-center bg-gray-400 dark:bg-gray-900 rounded-b-xl p-4 py-10 md:py-14 lg:py-16 transition duration-500 bg-opacity-70 dark:bg-opacity-70 min-h-auto"
  >
    <div
      className="flex flex-col justify-center gap-4 bg-gray-500 border-2 border-gray-300 rounded-lg p-4 mb-2 w-full sm:w-11/12 xl:w-2/3 md:h-56"
    >
      <div
        className=""
      >
        <div
          className="flex flex-col "
        >
          <h1
            className="text-2xl"
          >
            <div
              className="relative"
            >
              <a
                alt="company"
                experience-tool-tip-msg="Click here to visit the company's website"
                href="https://houston-inc.com/"
                target="blank"
              >
                Houston Inc
              </a>
            </div>
          </h1>
          <div
            className="text-md"
          >
            SEPTEMBER 2021 - PRESENT
          </div>
        </div>
        <div
          className="text-gray-900 pb-1 font-bold"
        >
          Software Developer - Consultant
        </div>
      </div>
      <p
        className="text-gray-300 border-t-2 border-gray-300 pt-3 text-justify"
      >
        Support to further scale, debug and maintain the application's functionality to our customers needs. From which I've had the opportunity to develope my skills and learn how to work with multiple technologies such as resources on AWS, DevOps with Docker and Kubernetes, GCloud, Auth0, Datadog and many more that are constantly used on our clients applications and integration. 
      </p>
    </div>
    <div
      className="flex flex-col justify-center gap-4 bg-gray-500 border-2 border-gray-300 rounded-lg p-4 mb-2 w-full sm:w-11/12 xl:w-2/3 md:h-56"
    >
      <div
        className=""
      >
        <div
          className="flex flex-col "
        >
          <h1
            className="text-2xl"
          >
            <div
              className="relative"
            >
              <a
                alt="company"
                experience-tool-tip-msg="Click here to visit the company's website"
                href="https://www.hotelkamp.com/en/"
                target="blank"
              >
                Hotel KÃ¤mp
              </a>
            </div>
          </h1>
          <div
            className="text-md"
          >
            SEPTEMBER 2015 - AUGUST 2020
          </div>
        </div>
        <div
          className="text-gray-900 pb-1 font-bold"
        >
          Room service and restaurant waiter
        </div>
      </div>
      <p
        className="text-gray-300 border-t-2 border-gray-300 pt-3 text-justify"
      >
        Responsible for the duties and maintenance of the room service department along with the guest's wishes and needs at the restaurant and guests rooms
      </p>
    </div>
    <div
      className="flex flex-col justify-center gap-4 bg-gray-500 border-2 border-gray-300 rounded-lg p-4 mb-2 w-full sm:w-11/12 xl:w-2/3 md:h-56"
    >
      <div
        className=""
      >
        <div
          className="flex flex-col "
        >
          <h1
            className="text-2xl"
          >
            Numerous restaurants and hotels
          </h1>
          <div
            className="text-md"
          >
            JANUARY 2007 - AUGUST 2020
          </div>
        </div>
        <div
          className="text-gray-900 pb-1 font-bold"
        >
          Waiter, Front Desk and kitchen staff
        </div>
      </div>
      <p
        className="text-gray-300 border-t-2 border-gray-300 pt-3 text-justify"
      >
        Waiter supervisor, cook, dishwasher. Experience that helps me to really listen and understand the customer's needs
      </p>
    </div>
    <div
      className="flex flex-col justify-center gap-4 bg-gray-500 border-2 border-gray-300 rounded-lg p-4 mb-2 w-full sm:w-11/12 xl:w-2/3 md:h-56"
    >
      <div
        className=""
      >
        <div
          className="flex flex-col "
        >
          <h1
            className="text-2xl"
          >
            Numerous bars, restaurants, concerts and music shops
          </h1>
          <div
            className="text-md"
          >
            JANUARY 1998 - AUGUST 2011
          </div>
        </div>
        <div
          className="text-gray-900 pb-1 font-bold"
        >
          Professional drummer/Drum teacher
        </div>
      </div>
      <p
        className="text-gray-300 border-t-2 border-gray-300 pt-3 text-justify"
      >
        Worked as a drummer and group leader in several bands and teaching drums as a side job
      </p>
    </div>
  </div>
</div>
`;
